qt/widgets/common/src/InterfaceManager.cpp

```diff
@@ -230,8 +233,11 @@ void InterfaceManager::registerHelpWindowFactory(
 
 MantidHelpInterface *InterfaceManager::createHelpWindow() const {
   if (m_helpViewer == nullptr) {
-    g_log.warning(
-        "Offline help is not available in this version of Workbench.");
+    if (!offlineHelpMsgDisplayed) {
+      g_log.information(
```

Comment:
Just to confirm: Your PR test says notice level. Did you intend for this to be information?

Reply:
Sorry, the PR is wrong, it's meant to be information as suggested by Nick in the issue. I'll update the PR